doc/requirements.rst

```diff
@@ -103,7 +103,6 @@ Design coverage
     :target: DESIGN
     :sourcetitle: Requirement
     :targettitle: Design
-    :nocaptions:
```

Comment:
Can you add test case with this?

Reply:
Added unit tests in a03147a86c18c44ccda970eaf584b2c0f5ae1c92